### PR TITLE
Rt 120862 no dot in default inc

### PR DESCRIPTION
**Pull request by jkeenan**

### CODE REVIEW

**Comment by nanis** [comment]
Thanks for noticing my oversight.  Much appreciated.

File: Makefile.PL [hunk 1]
```diff
@@ -8,6 +8,7 @@ use ExtUtils::MakeMaker;
 use File::Spec::Functions qw( catfile rel2abs );
 use Getopt::Long qw( GetOptionsFromArray );
 
+push @INC, '.';
 unless( require( catfile qw(inc IO Interactive Tiny.pm) ) ) {
```
Comment (kentfredric) [inline Makefile.PL]: Surely, the right thing to do here would be to augment the call to "require" to start with "./", as modifying `@INC` for the entirety of Makefile.PL is not actually necessary here.

**Comment by kentfredric** [comment]
Honestly, both this and #5 are ... well. Weird. Especially given the more obvious, safe, and logical answer alternative is:

```diff
diff --git a/Makefile.PL b/Makefile.PL
index fcc3a7d..937789a 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -8,7 +8,8 @@ use Getopt::Long qw( GetOptionsFromArray );
 use Path::Class;
 use Try::Tiny;
 
-use inc::IO::Interactive::Tiny;
+use lib 'inc';
+use IO::Interactive::Tiny;
 
 caller
     or run(\@ARGV, [qw{ssl crypto ssl32 ssleay32 eay32 libeay32 z}]);
-- 
2.13.1
```

Why all the special tricks?

**Comment by pedrohenriquecordeiro** [comment]
> Honestly, both this and #5 are ... well. Weird. Especially given the more obvious, safe, and logical answer alternative is:
> 
> ```diff
> diff --git a/Makefile.PL b/Makefile.PL
> index fcc3a7d..937789a 100644
> --- a/Makefile.PL
> +++ b/Makefile.PL
> @@ -8,7 +8,8 @@ use Getopt::Long qw( GetOptionsFromArray );
>  use Path::Class;
>  use Try::Tiny;
>  
> -use inc::IO::Interactive::Tiny;
> +use lib 'inc';
> +use IO::Interactive::Tiny;
>  
>  caller
>      or run(\@ARGV, [qw{ssl crypto ssl32 ssleay32 eay32 libeay32 z}]);
> -- 
> 2.13.1
> ```
> 
> Why all the special tricks?

this works for me, thanks

**Pull request merged**